docs/developing.md

```diff
@@ -0,0 +1,58 @@
+# Development guide
+
+This guide explains how to make changes to the `frontend-shared` package and use them with your local copy of the `client`.
+
+## Client setup
+
+Follow these steps to set up `frontend-shared` in the client repository.
+
+#### 1. Install yalc
+
+Install the `yalc` tool globally if you have not already done so.
+
+```shell
+yarn global add yalc
+```
+
+_Note_: [yalc](https://www.npmjs.com/package/yalc) is the tool that we use for locally making changes to a shared package and testing the changes in another project.
+
+#### 2. Build & publish
+
+Build the package and then publish to a a local yalc repository.
+
+```shell
+# from the frontend-shared dir
+$ yarn build
+$ yalc publish
+```
+
+#### 3. Link package
+
+In the `client` repository, link to you local pushed repository
```

Comment:
```suggestion
In the `client` repository, link to the package published with yalc
```

The link will point to what is effectively an extracted/installed package rather than a git repository.